examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<p>May be remove <code>:</code> at the end and add <code>.</code> instead?</p><p> </p><p>Also you can write <code>Let's check out model evaluation scores.</code>  if you want.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>I like the proposed rephrasing! regarding the usage of the colon instead of a period, I use it for all the instructional text as advised by Mike in the previous revisions.</p>